### DISCUSSION TITLE
Test for some existing caMicroscope functions

### DISCUSSION THREAD

**Discussion by We13b-MD** [camicroscope/caMicroscope#735] [General]
Hello everyone writing some tests for caMicroscope functions if you have any ideas let discuss it here

**Comment by We13b-MD** [comment]
I am having a bit of an issue writing  tests i think most functions are written in jquery if you have any ideas I am using jest by the way

**Reply by birm** [reply]
I think our code is eventually due for a refactor, and some components will certainly be easier to write tests for than others.

**Reply by We13b-MD** [reply]
@birm yes I noticed that, I will write tests for the other components hoping it will work fine thanks birm
